static/app/views/replays/detail/console/format.tsx

```diff
@@ -136,10 +139,27 @@ export default function Format({onExpand, expandPaths, args}: FormatProps) {
     } else {
       pieces.push(' ');
       pieces.push(
-        <ObjectInspector key={i} data={x} expandPaths={expandPaths} onExpand={onExpand} />
+        <Wrapper>
+          <StructuredEventData
+            key={i}
+            data={x}
+            initialExpandedPaths={expandPaths ?? []}
+            onToggleExpand={(expandedPaths, path) => {
+              onExpand(path, Object.fromEntries(expandedPaths.map(item => [item, true])));
```

Comment:
i wonder if we can extract this callback out into a constant, to reuse above too. 

like `const onToggleExpand = ...`

then here and above it can just be `onToggleExpand={onToggleExpand}`